resources/packaging/archlinux/PKGBUILD.in

```diff
@@ -30,11 +30,10 @@ source=(https://commondatastorage.googleapis.com/chromium-browser-official/chrom
         chromium-launcher-$_launcher_ver.tar.gz::https://github.com/foutrelis/chromium-launcher/archive/v$_launcher_ver.tar.gz
         chromium-$pkgver.txt::https://chromium.googlesource.com/chromium/src.git/+/$pkgver?format=TEXT
         #https://raw.githubusercontent.com/gcarq/inox-patchset/$pkgver/product_logo_{16,22,24,32,48,64,128,256}.png
-        # Patches
-$ungoog{patch_urls})
```

Comment:
If we're not going to need this and `patch_hashes`, we should probably cleanup the corresponding PKGBUILD generation code in buildkit.

Reply:
Yes, I will have a look at that. Let you know if I need help.